Build failure on windows

https://github.com/dyninst/dyninst/commit/3c38960b1e468579cdb62f67d5f123ea6c2463c8 broke the windows build.

C:\Users\Peter\Source\Repos\dyninst\common\src\addrtranslate-win.C(173): error C2440: 'return': cannot convert from 'HMODULE (__stdcall *)(LPCSTR,HANDLE,DWORD)' to 'Dyninst::Address'

Looks like AddressTranslateWin::getLibraryTrapAddrSysV needs to be changed, but I'm not sure what the correct fix is.

--- a/common/src/addrtranslate-win.C
+++ b/common/src/addrtranslate-win.C
@@ -171,5 +171,5 @@ AddressTranslateWin::AddressTranslateWin(PID pid, PROC_HANDLE phandle_) :
 
 Address AddressTranslateWin::getLibraryTrapAddrSysV()
 {
-	return &::LoadLibraryEx;
+	return reinterpret_cast<Address>(&::LoadLibraryEx);
 }

